src/pseudopeople/interface.py

```diff
@@ -301,9 +303,11 @@ def generate_current_population_survey(
         to a configuration YAML file, a configuration dictionary,
         or the sentinel value `pseudopeople.NO_NOISE`, which will generate a
         dataset without any configurable noise.
-    :param year: The survey date year (format YYYY) to include in the dataset. Will
-        raise a ValueError if there is no data for this year. If None is
-        provided, data from all years are included in the dataset.
+    :param year: The year for which to generate simulated Current Population Surveys of the simulated population (format YYYY, e.g., 2036);
+        the simulated dataset will contain records for surveys conducted on any date in the specified year.
+        Will
```

Comment:
Does your editor have a way to make these lines wrap better?  PEP-8 is picky about line length, so maybe the code standardizing tools the engineers use will do it...

Reply:
I can give it a shot -- might be kind of a pain to get everything to wrap properly

Reply:
I cleaned up the docstrings for census and ACS. Going to pause there so we can get this merged. I still need to edit the `state` parameter descriptions, so I can work on tidying up the wrapping as I work on that.